.circleci/config.yml

```diff
@@ -1,13 +1,15 @@
-####################
-# CircleCI configuration reference:
-#   https://circleci.com/docs/2.0/configuration-reference
-####################
+version: 2.1
 
-version: 2
+orbs:
+  gcp-gcr: circleci/gcp-gcr@0.16
+  docker: circleci/docker@2.6
+  python: circleci/python@2.1.1
+  gcp-cli: circleci/gcp-cli@3.2
 
-#####################################################
-# Jobs: see https://circleci.com/docs/2.0/jobs-steps/
-#####################################################
+executors:
+  ubuntu-machine-executor:
+    machine:
+      image: ubuntu-2004:current
```

Comment:
Ubuntu 20.04 reached end of standard support last month so I think this is a good opportunity to try `ubuntu-2404`.  Or just use `cimg/base:current` https://circleci.com/developer/images/image/cimg/base